src/stk/molecular/topology_graphs/topology_graph/optimizer/collapser.py

```diff
@@ -0,0 +1,177 @@
+"""
+Collapser
+=========
+
+"""
+
+from .optimizer import Optimizer
+from .utilities import (
+    get_mch_bond_topology,
+    merge_subunits,
+    OptimizationIncompleteError,
+)
+
+import mchammer as mch
+
+
+class Collapser(Optimizer):
+    """
+    Perform rigid-body collapse of molecule.
+
+    Implementation: [1]_
+
+    .. [1] https://github.com/andrewtarzia/MCHammer.
+
+    """
+
+    def __init__(
+        self,
+        step_size=0.1,
+        distance_threshold=1.5,
+        scale_steps=True,
+        save_trajectory=False,
+    ):
+        """
+        Initialize an instance of :class:`.Collapser`.
+
+        Parameters
+        ----------
+        step_size : :class:`float`, optional
+            The relative size of the step to take during collapse.
+            Defaults to 0.1 Angstrom.
```

Comment:
Do not duplicate defulat value in docstring.

Change to something like

```
The relative size of the step to take during collapse in Angstrom.
```

Reply:
Done.